digdag-tests/src/test/resources/acceptance/error/invalid.dig

```diff
@@ -0,0 +1,7 @@
++message:
+  echo>: Workflow started
++task:
+  echo>: ok
+  _error:
+    echo>: invalid
+    sh>: echo duplicated
```

Comment:
Considering https://github.com/treasure-data/digdag/pull/610#issuecomment-319573687, maybe we'd better add another test case that uses a nested `_error` with proper configuration and see if it finishes successfully.